packages/services/profusion/hello-world-service/api/removed.txt

```diff
@@ -0,0 +1 @@
+// Signature format: 2.0
```

Comment:
What are these api files? are they needed?

Reply:
They are used by the build system to detect changes. My build was complaining without them. The same happened in custom build of our client, which I had to update.

Reply:
But shouldn't they be in the `.gitignore`?

Reply:
No, they are being copied to the AOSP tree.